providers/tests/openlineage/plugins/test_listener.py

```diff
@@ -80,20 +79,30 @@ def regular_call(self, callable, callable_name, use_fork):
 def test_listener_does_not_change_task_instance(render_mock, xcom_push_mock):
     render_mock.return_value = render_df()
 
+    date = dt.datetime(2022, 1, 1)
```

Comment:
Nit: looks like start_date. Should we name it that way for explicity?

Reply:
It’s also used as logical date so I figured a more generic name is probably better.